instances/latest/terminologies/technique/multiPhotonFluorescenceMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/multiPhotonFluorescenceMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "multi photon fluorescence microscopy",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "multi photon microscopy"
```

Comment:
add: MPM

Reply:
And maybe others similar to ["2P microscopy"](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/technique/twoPhotonFluorescenceMicroscopy.jsonld)

Reply:
done